core/station/impl/src/models/blockchain_standard.rs

```diff
@@ -1,37 +1,98 @@
-use candid::CandidType;
 use orbit_essentials::storable;
 use std::{
     fmt::{Display, Formatter},
     str::FromStr,
 };
 
+use super::AddressFormat;
+
+#[storable]
+#[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct TokenStandardInfo {
+    pub name: String,
+    pub address_formats: Vec<AddressFormat>,
+}
+
 #[storable]
-#[derive(CandidType, Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
-pub enum BlockchainStandard {
-    Native,
+#[derive(Clone, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub enum TokenStandard {
+    InternetComputerNative,
     ICRC1,
-    ERC20,
+    // ERC20,
+}
+
+impl TokenStandard {
+    pub fn get_info(&self) -> TokenStandardInfo {
+        match self {
+            TokenStandard::InternetComputerNative => TokenStandardInfo {
+                name: "icp_native".to_owned(),
```

Comment:
I found it confusing since ICP is also the blockchain and a symbol. Since this is an arbitrary label I think it should reflect its meaning better. WDYT?

Reply:
Also, it used to be called `native`, not `icp`, I think renaming this to `icp_native` is better.

Reply:
I see, let's keep `icp_native` then